### DISCUSSION TITLE
Dream TV

### DISCUSSION THREAD

**Discussion by zupersponge** [iptv-org/community#705] [Q&A]
**Channel Name:** Dream TV
**Channel ID:** DreamTV.mn
**Country:** Mongolia
**Language:** Mongolian
**Website:** https://www.facebook.com/DreamTVMongolia/
**Notes (optional):**

**Comment by BellezaEmporium** [comment]
I'll need to check Facebook API to see if we can always get the last stream.

EDIT : They have a GraphQL... but hell x)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/mn.m3u#L8-L9
